src/assets/Generator.Shared/StringRequestContent.cs

```diff
@@ -19,12 +20,20 @@ public StringRequestContent(string value)
 
         public override async Task WriteToAsync(Stream stream, CancellationToken cancellation)
         {
+#if NET5_0_OR_GREATER || NETCOREAPP
```

Comment:
Should we just go with `NET6_0_OR_GREATER` here, since .NET 5 and .NET Core 3.1 are out of support?

Reply:
Can yes, is there plans to drop those now on the azure sdk?